### DISCUSSION TITLE
Request: annotated version of installed.jpg

### DISCUSSION THREAD

**Discussion by 1DontEx1st** [aderusha/DDD-Printable-Wall-Control-System#26] [Q&A]
I really like your layout, and while I don't have the same exact tools, I think it would be really helpful to know which parts you used for each section/tool of your layout. Is that something you'd be able to add?

**Comment by aderusha** [comment]
Moving this over to discussion!  Is there a specific question you have on something you're seeing?  There are a whole lot of parts and they all fit together in some way, so documenting every possible combo isn't always practical.  If you have a specific thing you're looking at and have questions on, let me know and I bet I can help you out.

**Reply by 1DontEx1st** [reply]
I'm partially just procrastinating from having to organize things...tools went on my boards a bit haphazardly, and I'm a recent wanna-be woodworker, so I don't even know _how_ I want things organized. I'll start with an easy one: are the hooks holding your drills/drivers/nailers/recip all the same U-Hook?

**Comment by mmohrmann** [comment]
I have the same questions about the picture. It would be cool to label what prints are holding up tools. What is holding up the tape measure? What is holding up the orbital sander?

**Comment by leviable** [comment]
I just discovered this project and was also confused about what exactly to do at first. While the stuff in README certainly helps guide one in the right direction, I still missed concepts like the pins and how spacers should be used. It would be helpful to have an example recipe prominently in the README to show exactly what is needed, and perhaps with an annotated picture for illustration?

## Example Recipe for 150mm calipers
* 1 [Caliper center piece](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/1x3%20Chinese%20Digital%20Calipers%20150mm.stl)
* 1 [2x1 L Bracket Left](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/L_brackets/2x1%20L%20Bracket%20Flat%20Left.stl)
* 1 [2x1 L Bracket Right](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/L_brackets/2x1%20L%20Bracket%20Flat%20Right.stl)
* 1 [2x3 Locking Spacer](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Locking_spacer/2x3%20Locking%20Spacer.stl)
* 1 [8mm Lock Pin](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Accessories/8mm%20Lock%20Pin.stl)
* 4 [10x8mm Pins](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Spacer_blank_flush/4x10x8mm%20Pin.stl)

**Reply by Gujustud** [reply]
Thank you so much! I was trying to figure out this exact Caliper setup, and it was the 10x8 pins that I was missing from the equation!

**Reply by terl22** [reply]
When the slots for the pins are oriented on the bottom, do you guys print with supports or will it generally be ok? For example this U-brackets has 7 recessed pin slots on the bottom
Sidepieces/U_brackets/2x3.25 2.5in U Bracket Left.stl

**Reply by aderusha** [reply]
All sidepieces should be oriented like that and the 4mm bridge over the top should print without any problems!

**Reply by terl22** [reply]
Excellent! One other question if you dont mind. For the side pieces, the print notes say use 100% perimeters for the catch area. I am wondering what is recommended, something like 10 perimeters looks like the maximum in my slicer, although when I printed the first test piece I got some thin areas on the catch point. Any setting advice for perimeter and infill % to get a solid print? Thanks!
![Screenshot 2023-12-11 155219](https://github.com/aderusha/DDD-Printable-Wall-Control-System/assets/153554649/10fb2b3f-c5bc-4b24-9dfb-7af66d1b40df)

![IMG_0068](https://github.com/aderusha/DDD-Printable-Wall-Control-System/assets/153554649/5fce66ee-3f17-40a5-92cc-a64c39232c72)

**Comment by HubbleTD19** [comment]
Downloaded the set and I'm starting to dig in. I'm struggling to understand the recipe for the dewalt chargers. The dcb095 is a 3x1, but it only has the top and bottom slots. The left and right 3x0 would work if they had pins, but they have slots too. What am I missing?

**Reply by akifbayram** [reply]
There are pins to use in those types of situations https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Accessories/4x10x8mm%20Pin.stl

**Reply by HubbleTD19** [reply]
Thanks! That makes those make a lot more sense now! I was picturing those at a wildly different scale.
